webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -113,29 +114,41 @@ export default {
     },
     cropImage() {
       this.showCropper = false
+      if (this.file.type === 'image/jpeg') {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-14T11:49:43Z" title="Friday, February 14th 2020, 12:49:43 pm +01:00">Feb 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -113,29 +114,41 @@ export default {
     },
     cropImage() {
       this.showCropper = false
+      if (this.file.type === 'image/png' || this.file.type === 'image/svg+xml') {
```


Hm, what about other file extensions? Couldn't you check for `jpg` only here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-14T11:56:44Z" title="Friday, February 14th 2020, 12:56:44 pm +01:00">Feb 14, 2020</time>_

---


I thought about that... I can change it